icu4j/main/tests/core/src/com/ibm/icu/dev/test/number/NumberFormatterApiTest.java

```diff
@@ -489,6 +489,45 @@ public void notationCompact() {
                 "Kun");
     }
 
+    @Test
+    public void unitUMinLocale() {
+        assertFormatSingleMeasure(
+                "Measure format take mu as celsius without usage",
+                "measure-unit/temperature-fahrenheit",
+                "unit/fahrenheit",
+                NumberFormatter.with().unit(MeasureUnit.FAHRENHEIT),
+                ULocale.forLanguageTag("en-US-u-mu-celsius"),
+                new Measure(0, MeasureUnit.FAHRENHEIT),
+                "0°F");
+
+        assertFormatSingleMeasure(
+                "Measure format for `en-US` take mu as celsius with usage",
+                "measure-unit/temperature-fahrenheit usage/default",
+                "unit/fahrenheit usage/default",
+                NumberFormatter.with().unit(MeasureUnit.FAHRENHEIT).usage("default"),
+                ULocale.forLanguageTag("en-US-u-mu-celsius"),
+                new Measure(0, MeasureUnit.FAHRENHEIT),
+                "-18°C");
```

Comment:
Please also test the outputUnit on the FormattedNumber.

Reply:
There is a bug, https://unicode-org.atlassian.net/browse/ICU-22154